benchmark_data_loading/bench_decoding.py

```diff
@@ -8,7 +8,7 @@
 from torchvision.io import decode_jpeg, ImageReadMode, read_file
 
 torch.set_num_threads(1)
-files = list((JPEG_FILES_ROOT / ("n01629819/" if args.tiny else "n01440764/")).glob("*.JPEG"))
+files = list((JPEG_FILES_ROOT / ("n01629819/" if args.tiny else "n01440764/")).rglob("*.JPEG"))
```

Comment:
At least on our internal cluster, the  images are in `n01629819/*.JPEG`.

```
(pt) ➜  benchmark_data_loading git:(benchmark_data_loading) ✗ ls /ontap_isolated/nicolashug/tinyimagenet/081318/train/n01629819/ | grep JPEG | wc -l
500
```

Will this still work in this setting?

Reply:
```
$ wget http://cs231n.stanford.edu/tiny-imagenet-200.zip
[...]
$ unzip tiny-imagenet-200.zip
$ tree -d tiny-imagenet-200
tiny-imagenet-200
├── test
│   └── images
├── train
│   ├── n01443537
│   │   └── images
│   ├── n01629819
│   │   └── images
│   ├── n01641577
│   │   └── images
[...]
└── val
    └── images
```

> Will this still work in this setting?

Yes. `rglob("*.JPEG")` is equivalent to `glob("**/*.JPEG")`. It just looks in every directory instead of just the top one. So unless on the cluster there are subfolders with more images, it should get you the same files.